.storybook/main.js

```diff
@@ -70,9 +78,11 @@ function overrideDefaultBabelLoader(rules) {
 
   const loader = /** @type {LoaderObjectDef}*/ (rule.use[loaderIdx]);
 
-  if (!Object.prototype.hasOwnProperty.call(loader, 'options')) {
+  if (loader && !Object.prototype.hasOwnProperty.call(loader, 'options')) {
```

Comment:
I think this check should be the other way, `if (!loader`

Reply:
This causes the same error posted above

Reply:
Right, that's because the config changed (what that `if` was intended to check for). Specifically based on the output you sent me, it looks like in the entries under `rule.use`, `test` is always empty and it's using full paths rather than names for `loader`. @Hotell any idea what should be done here? This is what `config.rules` looks like after the upgrade: https://hackmd.io/_bLkYEMbRpCCkUik5mn4Qg